phpstan-baseline.neon

```diff
@@ -208,7 +208,7 @@ parameters:
 
 		-
 			message: "#^Dead catch \\- PHPStan\\\\BetterReflection\\\\Identifier\\\\Exception\\\\InvalidIdentifierName is never thrown in the try block\\.$#"
-			count: 2
+			count: 3
```

Comment:
Not sure why it was ignored in this file so I added it here, but it looks reported correctly.